projects/driver_controls/src/fsm/power_fsm.c

```diff
@@ -18,41 +24,74 @@ FSM_DECLARE_STATE(state_off_brake);
 FSM_DECLARE_STATE(state_charging);
 FSM_DECLARE_STATE(state_charging_brake);
 FSM_DECLARE_STATE(state_on);
```

Comment:
Document: If we now have `state_on_brake` what is this state for? And why is there no `state_off_brake` I would expect symmetry here? Is the other case regen?

Reply:
We do have a `state_off_brake`? It's on line 23. Their purpose is documented on lines 8 to 12. I'll probably remove them once mech brake is designed.

Reply:
I've added a TODO marking the _brake FSMs to be removed.